projects/packages/connection/src/class-rest-connector.php

```diff
@@ -164,6 +165,8 @@ public function __construct( Manager $connection ) {
 				),
 			)
 		);
+
```

Comment:
Would it be worth adding a comment here to explain what this is? I don't know if that may be too much or really needed?

Reply:
@sergeymitr I think we could keep it simple for now and not create a separate class for token related endpoints. We can definitely do this in the future though, if we keep adding such endpoints!

Reply:
Makes sense, I'll merge this endpoint into `REST_Connector`.

Reply:
Done 👍